src/NServiceBus.Core.Tests/Persistence/InMemory/AnotherSagaWithUniquePropertyData.cs

```diff
@@ -3,37 +3,31 @@ namespace NServiceBus.SagaPersisters.InMemory.Tests
     using System;
     using System.Threading.Tasks;
 
-    class AnotherSagaWithTwoUniqueProperties:Saga<AnotherSagaWithTwoUniquePropertiesData>, IAmStartedByMessages<M1>
+    class AnotherSagaTwoUniqueProperty : Saga<AnotherSagaWithUniquePropertyData>, IAmStartedByMessages<M1>
     {
-        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<AnotherSagaWithTwoUniquePropertiesData> mapper)
+        public Task Handle(M1 message, IMessageHandlerContext context)
         {
-            mapper.ConfigureMapping<M1>(m => m.UniqueInt).ToSaga(s => s.UniqueInt);
-            mapper.ConfigureMapping<M1>(m => m.UniqueString).ToSaga(s => s.UniqueString);
+            throw new NotImplementedException();
         }
 
-        public Task Handle(M1 message, IMessageHandlerContext context)
+        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<AnotherSagaWithUniquePropertyData> mapper)
         {
-            throw new NotImplementedException();
+            mapper.ConfigureMapping<M1>(m => m.UniqueString).ToSaga(s => s.UniqueString);
         }
     }
 
     class M1
     {
         public string UniqueString { get; set; }
-
-        public int UniqueInt { get; set; }
     }
 
-    public class AnotherSagaWithTwoUniquePropertiesData : IContainSagaData
+    public class AnotherSagaWithUniquePropertyData : IContainSagaData
     {
+        public string UniqueString { get; set; }
```

Comment:
Sounds silly but can you move that down (after the properties from `IContainSagaData`?. More visible when you look at the saga data